spring-pulsar/src/main/java/org/springframework/pulsar/config/AbstractPulsarListenerContainerFactory.java

```diff
@@ -171,11 +175,27 @@ else if (this.autoStartup != null) {
 			instance.setAutoStartup(this.autoStartup);
```

Comment:
I noticed when looking at the copy properties that we have this 3rd way commented out above ...

```java
		// BeanUtils.copyProperties(this.containerProperties, properties, "topics",
		// "messageListener",
		// "batchListener", "subscriptionName", "subscriptionType", "schema");
```
we should probably yank this out. I can see though why that approach would be nice - we would not miss newly added props. The downside is. you can't do things like "set only if not set already for these fields but always copy these fields".